custom/icds_reports/utils/aggregation_helpers/ccs_record_monthly.py

```diff
@@ -157,25 +234,29 @@ def aggregation_query(self):
             ('closed', 'case_list.closed'),
             ('anc_abnormalities', 'agg_bp.anc_abnormalities'),
             ('home_visit_date', 'agg_bp.latest_time_end_processed'),
-            ('date_death', 'person.date_death'),
-            ('person_case_id', 'ucr.person_case_id')
+            ('date_death', 'person_cases.date_death'),
```

Comment:
I think you only want to add this if date_death is before month end

Reply:
Yeah correct